README.md

```diff
@@ -92,6 +94,23 @@ else fails, try removing the virtual environment and reinstalling:
 [local5000]:http://localhost:5000/
 
 
+### Troubleshooting Possible Errors
+
+Should you get series of type errors that looks something like `npm ERR! typeerror Error: Missing required argument #1`, after running `npm install`, this is most likely due to running an older version of node. 
+
+From experience, running Node versions 12+ seems to be okay. Follow [this](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) tutorial to upgrade your node version(for unix systems).
+
+Should you get an `OSError: [Errno 28] inotify watch limit reached` after running any command, this means that your system file watcher is running out of alloted handles, usually because the workspace is large and contains many files. 
+
+The solution is to run `sudo sysctl fs.inotify.max_user_watches=524288`
```

Comment:
Does this command change the limit permanently or only for the current session? I only knew one approach to permanently raising this limit, as suggested by [VS Code](https://code.visualstudio.com/docs/setup/linux#_visual-studio-code-is-unable-to-watch-for-file-changes-in-this-large-workspace-error-enospc).

Reply:
I think it's just for a session. But from my experience so far once you get past it while setting up, it doesn't pose an issue going further. Again, this is from personal experience.

Reply:
Noted